.circleci/config.yml

```diff
@@ -1,10 +1,10 @@
 version: 2.1
 orbs:
-  snyk: snyk/snyk@0.0.10
+  snyk: snyk/snyk@1.1.2
 jobs:
   build:
     docker:
-      - image: golang:1
+      - image: golang:1.17
```

Comment:
This will be reverted before merging.

Reply:
Testing with 1.18 locally works properly. This configuration should be changed to use the Golang CI orb in a follow-up.